File: frontend/src/pages/start.js
import React, { useState, useEffect } from 'react';
import { useNavigate } from 'react-router-dom';
import './startPage.css';

const StartPage = () => {
  const [selectedService, setSelectedService] = useState(0);
  const navigate = useNavigate(); 

  useEffect(() => {
    const timer = setInterval(() => {
      setSelectedService(0); 
    }, 2000);

    return () => clearInterval(timer);
  }, []);

  const handleGetStarted = () => {
    navigate('/choose'); 
  };

  return (
    <div className="start-page">
      <div className="logo-container">
        <img src="assets/quack.jpg" alt="Quack Logo" />
      </div>
      <div className="content-container">
        <div className="text-container">
          <h2>Easy, reliable way to take care of your home</h2>
        </div>
        <div className="text-container">
          <p>We provide you with the best people to help take care of your home.</p>
        </div>
        <div className="button-container">
          <button onClick={handleGetStarted}>Get Started</button>
        </div>
      </div>
    </div>
  );
};

export default StartPage;

File: old/start.dart
import 'dart:async';
import 'package:quack/pages/choose.dart';
import 'package:flutter/material.dart';

class StartPage extends StatefulWidget {
  const StartPage({Key? key}) : super(key: key);

  @override
  _StartPageState createState() => _StartPageState();
}

class _StartPageState extends State<StartPage> {
  int selectedService = 0;

  @override
  void initState() {
    // Randomly select from service list every 2 seconds
    Timer.periodic(Duration(seconds: 2), (timer) {
      setState(() {
        selectedService = 0; // Replace with your desired index or logic
      });
    });

    super.initState();
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      backgroundColor: Colors.white,
      body: Column(
        children: [
          SizedBox(
            height: 100,
          ),
          Container(
            height: MediaQuery.of(context).size.height * 0.45,
            width: MediaQuery.of(context).size.width,
            child: Center(
              child: Image.asset(
                'assets/quack.jpg', // Replace with your logo asset path
                height: 200, // Adjust the height as needed
              ),
            ),
          ),
          Expanded(
            child: Container(
              decoration: BoxDecoration(
                color: Color.fromARGB(160, 244, 244,
                    244), //////////////////////////////////////////////////////////
                borderRadius: BorderRadius.only(
                  topLeft: Radius.circular(80),
                  topRight: Radius.circular(80),
                ),
              ),
              child: Column(
                children: [
                  SizedBox(
                    height: 50,
                  ),
                  Container(
                    padding: EdgeInsets.symmetric(horizontal: 40),
                    child: Center(
                      child: Text(
                        'Easy, reliable way to take \ncare of your home',
                        textAlign: TextAlign.center,
                        style: TextStyle(
                          fontSize: 24,
                          fontWeight: FontWeight.bold,
                          color: Colors.grey.shade900,
                        ),
                      ),
                    ),
                  ),
                  SizedBox(
                    height: 20,
                  ),
                  Container(
                    padding: EdgeInsets.symmetric(horizontal: 60),
                    child: Center(
                      child: Text(
                        'We provide you with the best people to help take care of your home.',
                        textAlign: TextAlign.center,
                        style: TextStyle(
                          fontSize: 16,
                          color: Colors.grey.shade600,
                        ),
                      ),
                    ),
                  ),
                  Padding(
                    padding: EdgeInsets.all(50.0),
                    child: MaterialButton(
                      elevation: 0,
                      color: Colors.black,
                      onPressed: () {
                        Navigator.push(
                          context,
                          MaterialPageRoute(
                            builder: (context) => ChoosePage(),
                          ),
                        );
                      },
                      height: 55,
                      shape: RoundedRectangleBorder(
                        borderRadius: BorderRadius.circular(10),
                      ),
                      child: Center(
                        child: Text(
                          'Get Started',
                          style: TextStyle(
                            color: Colors.white,
                            fontSize: 18,
                            fontWeight: FontWeight.w500,
                          ),
                        ),
                      ),
                    ),
                  ),
                ],
              ),
            ),
          )
        ],
      ),
    );
  }
}
